app/controllers/appliances_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
 
   # GET /appliances/new
   def new
-    @appliance = Appliance.new
+    @appliance = Appliance.new(params.permit(:kitchen_id))
```

Comment:
Making it possible to auto-fill in advance the kitchen field when creating new appliance.